src/main/java/org/jenkinsci/plugins/pagerduty/changeevents/ChangeEventSender.java

```diff
@@ -36,6 +36,22 @@ public final void send(String integrationKey, Run<?, ?> build, TaskListener list
             e.printStackTrace(listener.error(e.getMessage()));
         }
     }
+    
+    public final void send(String integrationKey, String summaryText, Run<?, ?> build, TaskListener listener) {
+        try {
+            ChangeEvent changeEvent = getChangeEvent(integrationKey, summaryText,build);
```

Comment:
@uchinna Looking at this function and the original (without summaryText), I think we can eliminate a lot of redundant code by adding `summaryText` to the original function.  (Sorry, I know this is a change from what I said earlier regarding maintaining backwards compatibility).  I would suggest deleting the original `send()` method, as well as the original `getChangeEvent()` method and add the following line immediately after creating the `changeEvent`

```
if (summaryText != null) {
  //Override the default summary field
  changeEvent.setSummary(summaryText)
}
```